tests/services/test_crash_reports.py

```diff
@@ -0,0 +1,63 @@
+import glob
+import shutil
+
+import pytest
+
+from pymobiledevice3.services.crash_reports import CrashReportsManager
+
+
+TEST_DIR = 'temp'
+
+
+@pytest.fixture(scope='function')
+def crash_manager(lockdown):
+    return CrashReportsManager(lockdown)
+
+
+def test_ls_default(crash_manager):
+    crash_manager.afc.makedirs(TEST_DIR)
+    assert '/temp' in crash_manager.ls()
+
+
+def test_ls_path(crash_manager):
+    crash_manager.afc.makedirs(f'{TEST_DIR}/{TEST_DIR}')
+    assert f'/{TEST_DIR}/{TEST_DIR}' in crash_manager.ls(path=f'/{TEST_DIR}')
+
+
+@pytest.mark.parametrize('depth', [2, 3, 4])
+def test_ls_depth(crash_manager, depth):
+    path = ''
+    path_list = []
+    for i in range(depth):
```

Comment:
```python
path_component = f'/{TEST_DIR}'
path = [path_component] * depth
crash_manager.afc.makedirs(''.join(full_path))
```

Reply:
but still I need the path_list...

changed to something similar:

```
path = PATH_COMPONENT * depth
path_list = [PATH_COMPONENT * i for i in range(1, depth + 1)]
```